tests/utils/insert-block-using-slash.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * External dependencies
+ */
+import { canvas, insertBlock } from '@wordpress/e2e-test-utils';
+
+export const insertBlockUsingSlash = async ( blockTitle: string ) => {
+	await insertBlock( 'Paragraph' );
+	await canvas().keyboard.type( `/${ blockTitle }` );
```

Comment:
I suggest either waiting until the dropdown shows the block we are aiming for, or at least adding a `delay` option to the typing, so that it gives the dropdown time to react to that.

When pairing with @danieldudzic some days ago, I saw that, for example, sometimes the keyboard was too fast to type `admin/password` and was failing and retrying. This could potentially happen in this function too. So I suggest waiting for the element to be available.

Reply:
Fixed in 5bd0b708052c5fa1cb8490dd901a7d2aa10c8792